ERCS/erc-7632.md

```diff
@@ -0,0 +1,60 @@
+---
+eip: 7632
+title: Interfaces for Named Token
+description: Extend ERC-721 / ERC-1155 to support name in string and tokenId conversion.
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/18550
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-tbd-named-nfts-extending-erc-721/18550
```

You have to use the long url, though I could see an argument for permitting both forms.